modules/govuk/templates/static_extra_nginx_config.conf.erb

```diff
@@ -29,3 +29,17 @@ location ~ ^/media/ {
   # stack.
   include /etc/nginx/add-sts.conf;
 }
+
+location ~ ^/government/uploads/system/uploads/organisation/logo/ {
+  proxy_set_header Host <%= @asset_manager_host %>;
+  proxy_set_header X-Real-IP $remote_addr;
+  proxy_set_header X-Forwarded-Server $host;
+  proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+  proxy_set_header X-Forwarded-Host $host;
+  proxy_pass <%= @enable_ssl ? 'https' : 'http' %>://<%= @asset_manager_host %>;
+
+  # Explicitly re-include Strict-Transport-Security header, this
+  # forces nginx not to clear Cache-Control headers further up the
+  # stack.
+  include /etc/nginx/add-sts.conf;
+}
```

Comment:
> * Strict-Transport-Security
>   * Not present from Whitehall

Although this header is *not* present in responses served by Whitehall on the development VM, I believe [it *is* (or at least was) present in responses served by Whitehall in production][1].

[1]: https://github.com/alphagov/asset-manager/issues/236#issuecomment-333109045